src/main/resources/logback.xml

```diff
@@ -0,0 +1,6 @@
+<configuration scan="true">
+	<property scope="local" name="app.context.name" value="student-report-processor" />
+	<property scope="local" name="app.base.package.name" value="org.opentestsystem.delivery.studentreportprocessor" />
+	<property scope="local" name="mna.appender.active" value="true" />

```

Comment:
MNA represents a monitoring and altering app that this will basically replace.  Does this need to be set?  In other apps, I believe it is a ProgMan value